File: app/src/main/java/models/Person.java
package models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Felip on 13/06/2017.
 */

public class Person {

    @SerializedName("name")
    private String name;

    @SerializedName("surname")
    private String surname;

    @SerializedName("gender")
    private String gender;

    @SerializedName("region")
    private String region;

    @SerializedName("age")
    private String age;

    @SerializedName("phone")
    private String phone;

    @SerializedName("photo")
    private String photo;

    public String getFullName() {
        return name + " " + surname;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSurname() {
        return surname;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }
}

File: app/src/main/java/models/Person.kt
package models

import com.google.gson.annotations.SerializedName

/**
 * Created by Felip on 13/06/2017.
 */

class Person {

    @SerializedName("name")
    var name: String? = null

    @SerializedName("surname")
    var surname: String? = null

    @SerializedName("gender")
    var gender: String? = null

    @SerializedName("region")
    var region: String? = null

    @SerializedName("age")
    var age: String? = null

    @SerializedName("phone")
    var phone: String? = null

    @SerializedName("photo")
    var photo: String? = null
}